linux_os/guide/system/crypto_policies/configure_ssh_crypto_policy/ansible/shared.yml

```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_fedora
+# reboot = true
+# strategy = disable
+# complexity = low
+# disruption = medium
+
+- name: "@RULE_TITLE@"
+  lineinfile:
+    dest: /etc/sysconfig/sshd
+    state: absent
```

Comment:
@matusmarhefka I believe that this should be `state: present`

Reply:
@redhatrises Definitely not, if this regexp would be present in the `/etc/sysconfig/sshd` it would mean that sshd would opt out from the system-wide crypto policy which is exactly what this rule should prevent.